ntbs-service/Models/Validations/ValidationHelper.cs

```diff
@@ -38,7 +39,7 @@ public static string ValidYearLaterThanBirthYear(string name, int birthYear)
         public const string NhsNumberLength = "{0} needs to be 10 digits long";
         public const string InvalidNhsNumber = "This {0} is not valid. Confirm you have entered it correctly";
         public const string FieldRequired = "{0} is a mandatory field";
-        public const string PostcodeIsNotValid = "Postcode is not valid";
+        public const string PostcodeNotFound = "{0} is not found";
```

Comment:
As this is a super generic error message, should we just call it `NotFound` similarly to `NotValid`?

Reply:
I guess I just didn't expect this to be used for anything else, unlike the `not valid` and `please select` ones...

Reply:
I've also noted a method not being async in the repo and fixed that while there, so probably worth having another quick review, feel free to merge after that